staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/testing/v2/kms_plugin_mock.go

```diff
@@ -84,8 +82,8 @@ func WaitForBase64PluginToBeUp(plugin *Base64Plugin) error {
 		return gRPCErr == nil && resp.Healthz == "ok", nil
 	})
 
-	if pollErr == wait.ErrWaitTimeout {
-		return fmt.Errorf("failed to start kms-plugin, error: %v", gRPCErr)
+	if pollErr != nil {
```

Comment:
Let's make this change in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/testing/v2/kms_plugin_mock.go as well!

Reply:
I think you meant v1beta1. done!